java/dagger/internal/codegen/BindingGraph.java

```diff
@@ -1120,6 +1118,23 @@ private boolean hasLocallyPresentOptionalBinding(ResolvedBindings resolvedBindin
               && !getLocalExplicitBindings(keyFactory.unwrapOptional(resolvedBindings.key()).get())
                   .isEmpty();
         }
+
+        /**
+         * Returns {@code true} if {@code resolvedBindings} contains a binding for which there is
+         * an explicit present binding in this component that was resolved previously.
```

Comment:
"present" refers to optional bindings, as above. I don't think it makes sense here.